app/views/fragments/promotion/guardianWeeklyLegalTerms.scala.html

```diff
@@ -3,23 +3,36 @@
 @import com.gu.memsub.promo.Promotion.AnyPromotion
 @import com.gu.memsub.subsv2.Catalog
 @import views.support.MarkdownRenderer
+@import com.gu.i18n.{Country, CountryGroup}
 
-@(promoCode: PromoCode, catalog: Catalog, promotion: AnyPromotion, md: MarkdownRenderer)
+@(promoCode: PromoCode, catalog: Catalog, promotion: AnyPromotion, md: MarkdownRenderer, maybeCountry: Option[Country])
 @isSixForSix = @{(catalog.weekly.plans.flatten.filter(_.charges.billingPeriod == SixWeeks).map(_.id).toSet intersect promotion.appliesTo.productRatePlanIds).nonEmpty}
+
+@ukSixForSixTerms = @{"**United Kingdom:** Offer is £6 for the first 6 issues followed by quarterly (13 weeks) subscription payments of £37.50 thereafter, saving 35% off the cover price."}
+@usSixForSixTerms = @{"**USA:** Offer is $6 for the first 6 issues, followed by quarterly (13 weeks) subscription payments of $75, saving 31% off the cover price."}
+@caSixForSixTerms = @{"**Canada:** Offer is $6 for the first 6 issues, followed by quarterly (13 weeks) subscription payments of $80, saving 16% off the cover price."}
+@auSixForSixTerms = @{"**Australia:** Offer is $6 for the first 6 issues, followed by quarterly (13 weeks) subscription payments of $97.50 thereafter, saving 15% off the cover price."}
+@nzSixForSixTerms = @{"**New Zealand:** Offer is $6 for the first 6 issues, followed by quarterly (13 weeks) subscription payments of $123 thereafter, saving 20% off the cover price."}
+@euSixForSixTerms = @{"**Europe:**  Offer is €6 for the first 6 issues followed by quarterly (13 weeks) subscription payments of €61.30, saving 26% off the cover price. Excluding Cyprus, Malta and Republic of Ireland."}
+@rowSixForSixTerms = @{"**Rest of World:** Offer is US$6 the first 6 issues, followed by quarterly (13 weeks) subscription payments of US$81.30."}
+
+@sixForSixGenericTerms = @{
+    ukSixForSixTerms.concat("\n\n") +
+    usSixForSixTerms.concat("\n\n") +
+    caSixForSixTerms.concat("\n\n") +
+    auSixForSixTerms.concat("\n\n") +
+    nzSixForSixTerms.concat("\n\n") +
+    euSixForSixTerms.concat("\n\n") +
+    rowSixForSixTerms.concat("\n\n")
+}
+
 @Html(md.render(
```

Comment:
In case you're curious about this, I had planned to display the appropriate territory's terms here, but due to not knowing for sure which territory will be chosen, we have to show everything. Also, due to changing prices and discount percentages in regions that were previously expressed on one line (e.g. US and Canada, Aus and NZ) we thought it would be clearer to separate those out into their own lines.

However, if we do decide to be more selective in future we could do this;

```
@countryGroup = @{maybeCountry.flatMap( c =>
    CountryGroup.allGroups.find(_.countries.exists(_.alpha2 == c.alpha2))
)}

@sixForSixRegionalTerms = @{countryGroup.map{ cg =>
    cg.id.toLowerCase match {
        case "uk" => ukSixForSixTerms
        case "us" => usSixForSixTerms
        case "ca" => caSixForSixTerms
        case "au" => auSixForSixTerms
        case "nz" => nzSixForSixTerms
        case "eu" => euSixForSixTerms
        case _ => rowSixForSixTerms
    }
}}
```
and then say `{sixForSixRegionalTerms.getOrElse(sixForSixGenericTerms)}` in the render block.